app/helpers/library_hours_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LibraryHoursHelper
```

Comment:
Delete this file

Reply:
I've added this as a helper for now, although you mentioned doing this as a concern instead.